roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -16,8 +16,14 @@ coredns_ordinal_suffix: ""
 coredns_deployment_nodeselector: "kubernetes.io/os: linux"
 coredns_default_zone_cache_block: |
   cache 30
+
 coredns_host_network: false
 coredns_port: 53
+
+coredns_pod_disruption_budget: false
+# value for coredns pdb
+coredns_pod_disruption_budget_max_unavailable: 1
```

Comment:
This should probably be something like "30%" instead. 1 will slow down upgrade a lot in big clusters using dns-autoscaler.